src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.ts

```diff
@@ -89,4 +94,28 @@ export class PagesWalletsMyWalletsComponent implements OnInit {
 
     return dialogRef.afterClosed();
   }
+
+  public handleWalletDelete(wallet: MyWallet): void {
+
+    this.confirmDialogService.openConfirmModal({
+      headerText: `Deleting ${wallet.name} wallet`,
+      confirmationText: `Are you sure you want to delete ${wallet.name} wallet and all related data?`,
+      confirmBtnText: 'Yes',
```

Comment:
We can make "Yes" / "No" as default values for those text and not provided them :)